test/verifiedTests/analysis/testComputeFluxSplits/testcomputeFluxSplits.m

```diff
@@ -0,0 +1,66 @@
+% The COBRAToolbox: testcomputeFluxSplits.m
+%
+% Purpose:
+%     - tests the basic functionality of computeFluxSplits
+%       Tests 1 optional input with 3 possibilities: coeffSign=[] or 0 or 1
+%       returns 1 if all tests were completed succesfully, 0 if not
+%
+% Authors:
+%     - Original file: Diana El Assal 03/08/2017
+%
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testComputeFluxSplits'));
```

Comment:
@dayena, thanks for your test. There is an error:
```
================================================================================
Error occurred in testcomputeFluxSplits/testcomputeFluxSplits and it did not run to completion.

    ---------
    Error ID:
    ---------
    'MATLAB:load:couldNotReadFile'

    --------------
    Error Details:
    --------------
    Error using load
    Unable to read file 'testComputeFluxSplitsData.mat'. No such file or directory.
    
    Error in testcomputeFluxSplits (line 23)
    load('testComputeFluxSplitsData.mat');
================================================================================
```

It might be related to this line. May you try investigating by changing this line
```
fileDir = fileparts(which('testcomputeFluxSplits'));
```